app_widgetbook/lib/core/user/display_name.dart

```diff
@@ -6,10 +6,29 @@ import 'package:widgetbook_annotation/widgetbook_annotation.dart';
 @WidgetbookUseCase(name: 'Default', type: DisplayName)
 Widget displayNameDefaultUseCase(BuildContext context) {
   return DisplayName(
-    value: context.knobs.text(
-      label: 'Value',
-      description: 'Enter display name',
-      initialValue: 'John Doe',
+    user: User(
+      displayName: context.knobs.text(
+        label: 'Value',
+        description: 'Enter display name',
+        initialValue: 'John Doe',
+      ),
+      id: '',
+      username: '',
+    ),
+  );
+}
+
+@WidgetbookUseCase(name: 'Active', type: DisplayName)
+Widget displayNameActiveUseCase(BuildContext context) {
+  return DisplayName.active(
+    user: User(
+      displayName: context.knobs.text(
+        label: 'Value',
+        description: 'Enter display name',
+        initialValue: 'John Doe',
+      ),
+      id: '',
+      username: '',
     ),
   );
 }
```

Comment:
What's the difference between these two use-cases?

Reply:
Ok figured it out. It's the hover effect. Would be great to have an option within widgetbook to enable hover, but I'm not exactly sure how we can enable such functionality.

Reply:
Yes that's a great idea. 
For this specific component I think we'd have to expose the `isHovered` variable. But I think you mean have a general `Hovered` property which might be achieved in some way with a `MouseRegion` widget around the frame. I'll add it to the features ideas list 👍